docs/src/main/asciidoc/sqs.adoc

```diff
@@ -955,6 +957,8 @@ MessageListenerContainer<Object> listenerContainer(SqsAsyncClient sqsAsyncClient
 }
 ----
 
+NOTE: You can also specify the `SmartLifecycle.phase`, using `SqsMessageListenerContainer.builder()`, if you are interested on overriding the default value defined in `MessageListenerContainer.DEFAULT_PHASE`
```

Comment:
Same comment here about using `you`. Please try to rewrite using the passive voice.

Reply:
Done